src/components/Breadcrumbs/__stories__/index.stories.mdx

```diff
@@ -84,23 +54,21 @@ import Breadcrumbs, { breadcrumbsVariants } from '..'
 </Description>
 
 <Canvas>
-  <Story
-    name="Breadcrumbs-active"
-    args={{
-      activeIndex: 0,
-      variant: 'bubble',
-    }}
-  >
-    {args => (
-      <Breadcrumbs {...args}>
-        <Breadcrumbs.Item>Step 1</Breadcrumbs.Item>
-        <Breadcrumbs.Item>Step 2</Breadcrumbs.Item>
-        <Breadcrumbs.Item>Step 3</Breadcrumbs.Item>
-      </Breadcrumbs>
-    )}
+  <Story name="Active Index">
+    <Breadcrumbs activeIndex={1} variant="bubble">
+      <Breadcrumbs.Item>Step 1</Breadcrumbs.Item>
+      <Breadcrumbs.Item>Step 2</Breadcrumbs.Item>
+      <Breadcrumbs.Item>Step 3</Breadcrumbs.Item>
+    </Breadcrumbs>
   </Story>
 </Canvas>
 
+## API
+
+### `<Breadcrumbs>`
+
 <ArgsTable of={Breadcrumbs} />
 
-<ArgsTable of={Breadcrumbs.Item} />
+### `<Breadcrumbs.Item>`
+
+<ArgsTable of={Item} />
```

Comment:
Because if you use `Breadcrumbs.Item` it will not be able to infer the propTypes ?

Reply:
Exactly, like I did for estimated cost